app/models/account_user.rb

```diff
@@ -0,0 +1,3 @@
+class AccountUser < ActiveRecord::Base
+  enum role: [:user, :admin]
```

Comment:
Yes. It is an enum, not an AR association. The role field in the database is an integer, and this is an AR shortcut to set that integer to one of a defined list of values (in our case, :user and :admin).